code/game/objects/items/weapons/implants/implantpad.dm

```diff
@@ -1,85 +1,61 @@
-//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:32
-
 /obj/item/weapon/implantpad

```

Comment:
Are you sure this implantpad code is an actual improvement over what we currently have? I refactored the implantpad code a while back, and from the looks of it this reverts it back to an older state, which excludes things such as proper garbage collection, a remove implant verb, etc.